database/seed.js

```diff
@@ -0,0 +1,87 @@
+const schema = require('./index.js');
+
+// drop collections
+
+const randomType = () => {
+  const types = ['5/1 Arm', '30 Year Fixed', '15 Year Fixed'];
+  const index = Math.floor(Math.random() * 3);
+  return types[index];
+};
+
+const randomSeller = () => {
+  const banks = [['Aurora Financial, LLC', 6606],
+    ['NBKC Bank', 6706],
+    ['Sebonic Financial', 23606],
+    ['Homeplus Corporation', 34567],
+    ['New American Funding', 6606],
+    ['BNC National Bank', 418467]];
+
+  const index = Math.floor(Math.random() * 6);
```

Comment:
not major, but perhaps do banks.length instead of hardcoding 6 just in case you want to add more sample data to the seed